extension/test/unit/payment-handler.spec.js

```diff
@@ -0,0 +1,131 @@
+    }
+  }
+  /* eslint-enable max-len */
+
+  beforeEach(() => {
+    scope = nock(`${config.adyen.apiBaseUrl}`)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  it('when payment contains "submitPaymentDetailsRequest" with "makePaymentResponse", ' +
+    'then it should call "submitPaymentDetailsRequest"', async () => {
+    scope.post('/payments/details')
+      .reply(200, submitPaymentDetailsChallengeResponse)
+
+    const ctpPaymentClone = _.cloneDeep(ctpPayment)
+    ctpPaymentClone.custom.fields.makePaymentResponse = JSON.stringify(makePaymentRedirectResponse)
+    ctpPaymentClone.custom.fields.submitAdditionalPaymentDetailsRequest = JSON.stringify(submitPaymentDetailsRequest)
+
+    const response = await handlePayment(ctpPaymentClone)
+
+    expect(response.data.actions).to.have.lengthOf.above(0)
```

Comment:
that type of assertions does not have any clue, it could be anything but code will pass 😉 could you please consider updating the assertions like that please, I will not write for other ones in this file because it's obvious.

Reply:
You are right. In this test it doesn't matter what actions are being returned. This is tested in other tests like https://github.com/commercetools/commercetools-adyen-integration/pull/271/files/86d1d1156eac7bc65958344bda53bb36694e60e2#diff-2f7b0026dab2a37eb3a856341371ecdaR32

What is important here is that it returns some actions.